src/docs/cookbook/testing/integration/introduction.md

```diff
@@ -149,37 +149,37 @@ Now you can write tests. This involves four steps:
   2. Interact and tests widgets using the `WidgetTester` class.
   3. Test the important scenarios.
 
-<?code-excerpt "integration_test/app_test.dart"?>
+<?code-excerpt "lib/integration_test/app_test.dart (IntegrationTest)"?>
 ```dart
 import 'package:flutter_test/flutter_test.dart';
 import 'package:integration_test/integration_test.dart';
 
-import 'package:integration_test_experiments/main.dart' as app;
+import 'package:introduction/main.dart' as app;
 
 void main() {
   IntegrationTestWidgetsFlutterBinding.ensureInitialized();
 
   group('end-to-end test', () {
     testWidgets('tap on the floating action button, verify counter',
-            (WidgetTester tester) async {
-          app.main();
-          await tester.pumpAndSettle();
+        (WidgetTester tester) async {
```

Comment:
Hi! Did you actually run the code formatter over this code? Or did you adjust it manually/visually?

Reply:
Hi Shams! I ran `dart format lib/integration_test/app_test.dart` since the snippet is being pulled in from that file. I didn't adjust any of the formatting myself. Just ran it again to be safe:

```
$ dart format lib/integration_test/app_test.dart 
Formatted 1 file (0 changed) in 0.19 seconds.
```

Reply:
+1!